### DISCUSSION TITLE
request for pure sniffer mode

### DISCUSSION THREAD

**Discussion by SSSlee** [emsesp/EMS-ESP32#1585] [Ideas]
Hi, would it be possible to add a sniffer mode for the EMS-Bus? I understand that this flag only blocks wirte commands, but read commands are still possible:
<img width="507" alt="image" src="https://github.com/emsesp/EMS-ESP32/assets/153392645/df7608a7-3784-417a-a909-c18bdfcbd47e">

My wish would be to have an mode of operation that has no influence on bus communication at all

**Comment by MichaelDvP** [comment]
In telnet (su) `set tx_mode 0` disables all TX to the bus.
But we do not get the version/product-id of the devices, so the devices list will stay empty. (Maybe the thermostat requests this info on changes, but not often).

**Reply by SSSlee** [reply]
> In telnet (su) set tx_mode 0 disables all TX to the bus.

Thanks, that's what I'm searching for. :-)

**Reply by proddy** [reply]
and I've added this to the documentation to make it more clear,
